packages/sanity/src/desk/comments/src/components/list/CommentsListItemLayout.tsx

```diff
@@ -65,6 +65,12 @@ const RetryCardButton = styled(Card)`
 
 const StyledCommentsListItemContextMenu = styled(CommentsListItemContextMenu)``
 
+// Container which displays comment dates in gray.
+// Transparent tone cards still contain a background, which we unset to ensure parent colors take precedence.
+const TransparentDateCard = styled(Card).attrs({tone: 'transparent'})`
+  background: unset;
```

Comment:
Is this really using 800 as tint value (as requested from design)? Should we access the color from `@sanity/color` instead – to really make sure that the color is always gray and has the correct tint?


```ts
import {hues} from '@sanity/color'

const TimeText = styled(Text)(({theme}) => {
  const isDark = theme.sanity.color.dark
  const fg = hues.gray[isDark ? 50 : 800].hex

  return css`
    min-width: max-content;
    --card-fg-color: ${fg};
  `
})

```

```ts
<TimeText size={0} />
```